pkg/login/auth_test.go

```diff
@@ -27,6 +27,7 @@ func TestAuthenticateUser(t *testing.T) {
 				So(sc.grafanaLoginWasCalled, ShouldBeFalse)
 				So(sc.ldapLoginWasCalled, ShouldBeFalse)
 				So(err, ShouldEqual, ErrPasswordEmpty)
+				So(sc.loginUserQuery.AuthModule, ShouldEqual, "")
```

Comment:
Is it important for this particular test to test for the value of the auth module?

Reply:
I think it helps to understand the behavior of the function for this field.